packages/salesforcedx-apex-debugger/src/commands/baseCommand.ts

```diff
@@ -14,19 +15,22 @@ export abstract class BaseCommand {
   private readonly accessToken: string;
   private readonly debuggedRequestId: string;
   private readonly debuggerApiPath = 'services/debug/v41.0';
```

Comment:
I know this has existed before this CL, could the number 41.0 be pulled into a constants file somewhere?

Reply:
Is this really necessary? This basically *is* the single constant used for all debugger api calls. We can update it here when necessary.

Are you thinking of some global constant class/interface for all code?

Reply:
Into a file called Constants.ts